app/scripts/metamask-controller.js

```diff
@@ -1799,9 +1798,7 @@ export default class MetamaskController extends EventEmitter {
           `${this.loggingController.name}:add`,
         ],
       }),
-      isEthSignEnabled: () =>
-        this.preferencesController.store.getState()
-          ?.disabledRpcMethodPreferences?.eth_sign,
+      isEthSignEnabled: false,
```

Comment:
just checking that this can be a constant and not a function.

Doesn't seem like it's used anywhere at all in the SignatureController though

https://github.com/MetaMask/core/blob/main/packages/signature-controller/src/SignatureController.ts#L114

Reply:
ooof yeah should have been removed in the SignatureController